public/app/plugins/panel/stat/common.ts

```diff
@@ -107,3 +112,55 @@ export function addOrientationOption<T extends SingleStatBaseOptions>(
     defaultValue: VizOrientation.Auto,
   });
 }
+
+export function formatDisplayValuesWithCustomUnits(
+  fieldValues: FieldDisplay[],
+  customUnits: { customPrefix: string; customSuffix: string }
```

Comment:
```suggestion
  config: PanelFieldConfig
```
can we just pass the options directly?

Reply:
Updated.